igneous/tasks/tasks.py

```diff
@@ -878,13 +879,21 @@ def __init__(
     self.fill_missing = fill_missing
     self.translate = Vec(*translate)
     self.mip = int(mip)
-    self.clip_fraction = float(clip_fraction)
+    if isinstance(clip_fraction, Sequence):
```

Comment:
Cool ,I didn't know about Sequence. A better way to test could be trying `iter(...)` since that would work for other types as well, though that's probably not necessary here.

Reply:
`Sequence` was chosen because I need it to be subscriptable and wanted to check the length, i.e. I used `__getitem()__` and `__len()__`.